src/scripts/site.js

```diff
@@ -478,11 +476,7 @@ var AppInfo = {};
 
         var preferNativeAlerts = browser.tv;
```

Comment:
this is probably not needed anymore

Reply:
If the client never enter in a condition when browser.tv is true, yes this is useless. But there are lots of code regarding browser.tv. This should be targeted in another PR to remove all browser.tv related code. What do you think? 

Even if we add future support to put this inside a tv, the default alert mechanism should work out of the box, since it's based on dom manipulation to achieve a dialog.

Reply:
I was referring to the `preferNativeAlerts` variable only (`browser.tv` is needed, I'm not arguing with it).

Reply:
Oh right, so looking into that, one could assume all clients support the non native method present here. We want that UI is equal to every client, so.. don't see the need to use nativeconfirm and nativeprompt here (window.confirm and window.prompt). What's your opinion? If you agree, i can remove nativeprompt and nativeconfirm, removing also preferNativeAlerts all together.

Reply:
I guess it was there for a reason, maybe native alerts/prompts on TVs could be more easy to navigate using remotes?.. maybe leave them in for now...

Reply:
sure np